### ISSUE TITLE
Avatar's visual content not communicated as text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-avatar#26]
### Issue
Ensure embedded elements and canvas elements provide a meaningful text equivalent

The canvas element used part of the profile image buttons do not provide any textual equivalents part of the canvas element. This is for both Initial and Image state. Button with function to add or change an image is present, but equivalent missing for the visual content itself. 

```
<canvas class="bb-avatar-initials" ng-show="$ctrl.showInitials()" width="100" height="100"></canvas>
```

### Expected Behavior

When the canvas element is used developers should provide fallback content within the canvas element's subtree.

It is recommended to add textual equivalents part of the canvas element. However, if aria-label is used on the container button element, the labeled canvas element can be hidden from screen readers using aria-hidden property.

[Code]
<canvas class="bb-avatar-initials" ng-show="$ctrl.showInitials()" width="100" height="100">
<p>Jarod Bonino</p>
</canvas>

[AB#1564779](https://blackbaud.visualstudio.com/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/1564779)

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-MattGregg the Sky 2+ avatar does not use `canvas` under the hood (the example here appears to be Sky 1). 

I think we probably still need to add some sort of text equivalent but just wanted to verify :) Does this just need to be an `aria-label` or something more than that?

https://github.com/blackbaud/skyux-avatar/blob/master/src/app/public/modules/avatar/avatar.inner.component.html

**Comment by Blackbaud-MattGregg** [comment]
@Blackbaud-TrevorBurch Yeah. Apologies I didn't catch that in the audit materials. For SKY 2+ avatar, it does look to have a similar issue. There is no text equivalent for the content and is semantically a button. Screen reader users just hear "Drag a file here or click to browse"
 
To make this accessible for screen reader users, I think it would need to be two elements somehow in there:

1. Content element w/ text equivalent of the content, so something like..
- if image - "Profile photo of Trevor Burch"
- if initials - <abbr>TB</abbr>
2. Button to "[Add|Change] profile photo of Trevor Burch. Drag a file here or click to browse."

**Comment by Blackbaud-TrevorBurch** [comment]
Make File Drop take in a custom aria-label and move contents above button element.

In avatar - pass custom aria-label to file drop and then update avatar inner component to have screen reader text for images. Initials appear to be fine today - verify with screen reader.

**Comment by Blackbaud-TrevorBurch** [comment]
In `4.1.1`
